j2cl:watch crashes when doing hot changes on windows 11

Hi, when i run j2cl:watch , then jetty:run, the program runs nicely but when i make changes in src packages, the j2cl:watch crashes, curiously when i make changes in the css folder, my IDE (VS Code) tells me i can't make any changes(maybe privilleges?)
i will put all the stacktrace
PS C:\workspaces\j2cl\piru> mvn j2cl:watch
[INFO] Scanning for projects...
[INFO] 
[INFO] -----------------------------< piru:piru >------------------------------
[INFO] Building piru 1.0-SNAPSHOT
[INFO] --------------------------------[ war ]---------------------------------
[INFO] 
[INFO] --- j2cl-maven-plugin:0.19:watch (default-cli) @ piru ---
[INFO] Using C:\workspaces\j2cl\piru\target/piru-1.0-SNAPSHOT as webappDirectory since goal is running in a non-reactor build and none was set
[INFO] Removing dependency com.google.jsinterop:base:jar:1.0.0:compile, replacing with com.vertispan.jsinterop:base:jar:1.0.0-1
[INFO] Removing dependency com.google.jsinterop:base:jar:1.0.0:compile, replacing with com.vertispan.jsinterop:base:jar:1.0.0-1
[INFO] Removing dependency com.google.jsinterop:base:jar:1.0.0:compile, replacing with com.vertispan.jsinterop:base:jar:1.0.0-1
[INFO] Removing dependency com.google.jsinterop:base:jar:1.0.0:compile, replacing with com.vertispan.jsinterop:base:jar:1.0.0-1
[INFO] Starting com.google.elemental2:elemental2-dom:1.1.0/unpack
[INFO] Finished com.google.elemental2:elemental2-dom:1.1.0/unpack in 328ms
[INFO] Starting com.google.jsinterop:jsinterop-annotations:2.0.0/unpack
[INFO] Starting com.google.elemental2:elemental2-core:1.1.0/unpack
[INFO] Finished com.google.elemental2:elemental2-core:1.1.0/unpack in 61ms
[INFO] Starting com.google.elemental2:elemental2-promise:1.1.0/unpack
[INFO] Starting com.vertispan.jsinterop:base:1.0.0-1/unpack
[INFO] Finished com.vertispan.jsinterop:base:1.0.0-1/unpack in 8ms
[WARNING] Test watch temporarily disabled
[INFO] Start watching {piru:piru:1.0-SNAPSHOT=[C:\workspaces\j2cl\piru\src\main\java]}
[INFO] Watching for changes
[INFO] Starting piru:piru:1.0-SNAPSHOT/jszipbundle
[INFO] Starting com.google.elemental2:elemental2-dom:1.1.0/bytecode
[INFO] Starting com.vertispan.jsinterop:base:1.0.0-1/bytecode
[INFO] Finished com.vertispan.jsinterop:base:1.0.0-1/bytecode in 11ms
[INFO] Starting com.google.jsinterop:jsinterop-annotations:2.0.0/bytecode
[INFO] Finished com.google.jsinterop:jsinterop-annotations:2.0.0/bytecode in 6ms
[INFO] Starting com.google.elemental2:elemental2-core:1.1.0/bytecode
[INFO] Finished com.google.elemental2:elemental2-core:1.1.0/bytecode in 90ms
[INFO] Starting com.google.elemental2:elemental2-promise:1.1.0/bytecode
[INFO] Finished com.google.elemental2:elemental2-promise:1.1.0/bytecode in 9ms
[INFO] Starting com.vertispan.jsinterop:base:1.0.0-1/stripped_sources
[INFO] Finished com.google.elemental2:elemental2-dom:1.1.0/bytecode in 659ms
[INFO] Finished com.vertispan.jsinterop:base:1.0.0-1/stripped_sources in 95ms
[INFO] Starting com.google.jsinterop:jsinterop-annotations:2.0.0/stripped_sources
[INFO] Starting com.google.elemental2:elemental2-core:1.1.0/stripped_sources
[INFO] Finished com.google.elemental2:elemental2-core:1.1.0/stripped_sources in 37ms
[INFO] Starting com.google.elemental2:elemental2-promise:1.1.0/stripped_sources
[INFO] Starting com.vertispan.jsinterop:base:1.0.0-1/transpiled_js
[INFO] Finished piru:piru:1.0-SNAPSHOT/jszipbundle in 2018ms
[INFO] Finished com.vertispan.jsinterop:base:1.0.0-1/transpiled_js in 1191ms
[INFO] Starting com.vertispan.jsinterop:base:1.0.0-1/stripped_bytecode
[INFO] Starting com.google.jsinterop:jsinterop-annotations:2.0.0/stripped_bytecode
[INFO] Starting com.google.jsinterop:jsinterop-annotations:2.0.0/transpiled_js
[INFO] Starting com.vertispan.jsinterop:base:1.0.0-1/bundled_js
[INFO] Finished com.vertispan.jsinterop:base:1.0.0-1/bundled_js in 131ms
[INFO] Starting com.google.jsinterop:jsinterop-annotations:2.0.0/stripped_bytecode_headers
[INFO] Starting com.google.jsinterop:jsinterop-annotations:2.0.0/bundled_js
[INFO] Finished com.vertispan.jsinterop:base:1.0.0-1/stripped_bytecode in 471ms
[INFO] Starting com.vertispan.jsinterop:base:1.0.0-1/stripped_bytecode_headers
[INFO] Starting com.google.elemental2:elemental2-promise:1.1.0/transpiled_js
[INFO] Starting com.google.elemental2:elemental2-promise:1.1.0/stripped_bytecode
[INFO] com.google.elemental2:elemental2-promise:1.1.0/stripped_bytecode: /C:/workspaces/j2cl/piru/target/gwt3BuildCache/0.19/com.google.elemental2-elemental2-promise-1.1.0/b9a9c421dcc5a5ad53af6450b3f7ae6a-stripped_sources/results/elemental2/promise/Promise.java C:\workspaces\j2cl\piru\target\gwt3BuildCache\0.19\com.google.elemental2-elemental2-promise-1.1.0\b9a9c421dcc5a5ad53af6450b3f7ae6a-stripped_sources\results\elemental2\promise\Promise.java uses unchecked or unsafe operations.
[INFO] com.google.elemental2:elemental2-promise:1.1.0/stripped_bytecode: /C:/workspaces/j2cl/piru/target/gwt3BuildCache/0.19/com.google.elemental2-elemental2-promise-1.1.0/b9a9c421dcc5a5ad53af6450b3f7ae6a-stripped_sources/results/elemental2/promise/Promise.java Recompile with -Xlint:unchecked for details.
[INFO] Finished com.google.elemental2:elemental2-promise:1.1.0/stripped_bytecode in 131ms
[INFO] Finished com.google.elemental2:elemental2-promise:1.1.0/transpiled_js in 136ms
[INFO] Starting com.google.elemental2:elemental2-promise:1.1.0/stripped_bytecode_headers
[INFO] Starting com.google.elemental2:elemental2-core:1.1.0/transpiled_js
[INFO] Starting com.google.elemental2:elemental2-core:1.1.0/stripped_bytecode
[INFO] com.google.elemental2:elemental2-core:1.1.0/stripped_bytecode: /C:/workspaces/j2cl/piru/target/gwt3BuildCache/0.19/com.google.elemental2-elemental2-core-1.1.0/4ee56d58cd973f29c315b0e308cfff8c-stripped_sources/results/elemental2/core/JsArray.java C:\workspaces\j2cl\piru\target\gwt3BuildCache\0.19\com.google.elemental2-elemental2-core-1.1.0\4ee56d58cd973f29c315b0e308cfff8c-stripped_sources\results\elemental2\core\JsArray.java uses unchecked or unsafe operations.
[INFO] com.google.elemental2:elemental2-core:1.1.0/stripped_bytecode: /C:/workspaces/j2cl/piru/target/gwt3BuildCache/0.19/com.google.elemental2-elemental2-core-1.1.0/4ee56d58cd973f29c315b0e308cfff8c-stripped_sources/results/elemental2/core/JsArray.java Recompile with -Xlint:unchecked for details.
[INFO] Finished com.google.elemental2:elemental2-core:1.1.0/stripped_bytecode in 656ms
[INFO] Finished com.google.elemental2:elemental2-core:1.1.0/transpiled_js in 1016ms
[INFO] Starting com.google.elemental2:elemental2-promise:1.1.0/bundled_js
[INFO] Finished com.google.elemental2:elemental2-promise:1.1.0/bundled_js in 53ms
[INFO] Starting com.google.elemental2:elemental2-core:1.1.0/stripped_bytecode_headers
[INFO] Finished com.google.elemental2:elemental2-core:1.1.0/stripped_bytecode_headers in 61ms
[INFO] Starting com.google.elemental2:elemental2-core:1.1.0/bundled_js
[INFO] Starting com.google.elemental2:elemental2-dom:1.1.0/stripped_sources
[INFO] Starting piru:piru:1.0-SNAPSHOT/bytecode
[INFO] Finished com.google.elemental2:elemental2-dom:1.1.0/stripped_sources in 194ms
[INFO] Finished piru:piru:1.0-SNAPSHOT/bytecode in 261ms
[INFO] Starting piru:piru:1.0-SNAPSHOT/stripped_sources
[INFO] Finished com.google.elemental2:elemental2-core:1.1.0/bundled_js in 1174ms
[INFO] Starting com.google.elemental2:elemental2-dom:1.1.0/stripped_bytecode
[INFO] Starting com.google.elemental2:elemental2-dom:1.1.0/transpiled_js
[INFO] com.google.elemental2:elemental2-dom:1.1.0/stripped_bytecode: /C:/workspaces/j2cl/piru/target/gwt3BuildCache/0.19/com.google.elemental2-elemental2-dom-1.1.0/667d0d24e90bada4d4a6a06318e31144-stripped_sources/results/elemental2/dom/DOMRectReadOnly.java Some input files use or override a deprecated API.
[INFO] com.google.elemental2:elemental2-dom:1.1.0/stripped_bytecode: /C:/workspaces/j2cl/piru/target/gwt3BuildCache/0.19/com.google.elemental2-elemental2-dom-1.1.0/667d0d24e90bada4d4a6a06318e31144-stripped_sources/results/elemental2/dom/DOMRectReadOnly.java Recompile with -Xlint:deprecation for details.  
[INFO] Finished com.google.elemental2:elemental2-dom:1.1.0/stripped_bytecode in 1670ms
[INFO] Finished com.google.elemental2:elemental2-dom:1.1.0/transpiled_js in 2373ms
[INFO] Starting com.google.elemental2:elemental2-dom:1.1.0/stripped_bytecode_headers
[INFO] Finished com.google.elemental2:elemental2-dom:1.1.0/stripped_bytecode_headers in 290ms
[INFO] Starting piru:piru:1.0-SNAPSHOT/transpiled_js
[INFO] Finished piru:piru:1.0-SNAPSHOT/transpiled_js in 51ms
[INFO] Starting piru:piru:1.0-SNAPSHOT/bundled_js
[INFO] Finished piru:piru:1.0-SNAPSHOT/bundled_js in 34ms
[INFO] Starting com.google.elemental2:elemental2-dom:1.1.0/bundled_js
[INFO] Finished com.google.elemental2:elemental2-dom:1.1.0/bundled_js in 6077ms
[INFO] Starting piru:piru:1.0-SNAPSHOT/bundled_js_app
[INFO] Starting final task piru:piru:1.0-SNAPSHOT/bundled_js_app
[INFO] Finished final task piru:piru:1.0-SNAPSHOT/bundled_js_app in 1233ms
[INFO] -----  Build Complete: ready for browser refresh  -----
[INFO] Starting piru:piru:1.0-SNAPSHOT/bytecode
[INFO] Finished piru:piru:1.0-SNAPSHOT/bytecode in 162ms
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.nio.file.FileSystemException: C:\workspaces\j2cl\piru\target\piru-1.0-SNAPSHOT\piru\piru.js: The requested operation cannot be performed on a file with a user-mapped section open
        at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:92)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
        at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
        at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:484)
        at java.base/java.nio.file.Files.newOutputStream(Files.java:228)
        at java.base/java.nio.file.Files.write(Files.java:3581)
        at java.base/java.nio.file.Files.write(Files.java:3633)
        at com.vertispan.j2cl.build.provided.BundleJarTask$1.finish(BundleJarTask.java:131)

--- a/j2cl-archetypes/j2cl-archetype-servlet/src/main/resources/archetype-resources/__rootArtifactId__-server/pom.xml
+++ b/j2cl-archetypes/j2cl-archetype-servlet/src/main/resources/archetype-resources/__rootArtifactId__-server/pom.xml
@@ -36,8 +36,10 @@
                         <scanIntervalSeconds>1</scanIntervalSeconds>
                         <webApp>
                             <extraClasspath>\${project.basedir}/../\${rootArtifactId}-shared/target/classes/</extraClasspath>
+                            <_initParams>
+                                <org.eclipse.jetty.servlet.Default.useFileMappedBuffer>false</org.eclipse.jetty.servlet.Default.useFileMappedBuffer>
+                            </_initParams>
                         </webApp>
-                        <contextXml>\${project.basedir}/src/main/jettyconf/context.xml</contextXml>
                     </configuration>
                 </plugin>
             </plugins>

--- a/j2cl-archetypes/j2cl-archetype-servlet/src/main/resources/archetype-resources/__rootArtifactId__-server/src/main/jettyconf/context.xml
+++ b/j2cl-archetypes/j2cl-archetype-servlet/src/main/resources/archetype-resources/__rootArtifactId__-server/src/main/jettyconf/context.xml
@@ -1,6 +1,0 @@
-<Configure class="org.eclipse.jetty.webapp.WebAppContext">
-    <Call name="setInitParameter">
-        <Arg>org.eclipse.jetty.servlet.Default.useFileMappedBuffer</Arg>
-        <Arg>false</Arg>
-    </Call>
-</Configure>

--- a/j2cl-archetypes/j2cl-archetype-simple/src/main/resources/archetype-resources/pom.xml
+++ b/j2cl-archetypes/j2cl-archetype-simple/src/main/resources/archetype-resources/pom.xml
@@ -107,6 +107,9 @@
         <version>\${maven.jetty.plugin.version}</version>
         <configuration>
           <webApp>
+            <_initParams>
+              <org.eclipse.jetty.servlet.Default.useFileMappedBuffer>false</org.eclipse.jetty.servlet.Default.useFileMappedBuffer>
+            </_initParams>
             <resourceBases>
               <!-- Serve static content without copying from src/main/webapp -->
               <resourceBase>src/main/webapp</resourceBase>

